hello-world/brysonsteck/.gitignore

```
@@ -0,0 +1,12 @@
+*.iml
+.gradle
+/local.properties
```

```suggestion
*.apk
*.jar
/local.properties
```

Be sure to exclude compiled binaries!